examples/visualization.ipynb

```diff
@@ -73,7 +77,7 @@
     "trainer, ret_dict = pl_val(yaml.load(cfg, Loader=yaml.Loader));\n",
     "\n",
     "clb_metric = [x for x in trainer.callbacks if isinstance(x, MetricValCallback)][0]\n",
-    "vis = RetrievalVisualizer.from_embeddings_metric(clb_metric.metric)\n"
+    "# vis = RetrievalVisualizer.from_embeddings_metric(clb_metric.metric)  # todo: fix in next pr\n"
```

Comment:
Remove todo?

Reply:
I mean, if we will fix it later why do we need this todo in master repo at all?

Reply:
I prefer to keep it as a reminder